1-exercises/A-array-find/exercise.js

```diff
@@ -16,6 +16,11 @@ let names = [
   "Karim",
   "Ahmed",
 ];
+function findLongNameThatStartsWithA(names) {
+  names.sort((a, b) => b.length - a.length);
+  let longestName = names.find((name) => name[0] === "A");
+  return longestName;
+}
 
```

Comment:
I like your solutions however there is another solution that you can try with 'find()' method